### PR TITLE
Added Check for empty messages

### PR DESCRIPTION
**Pull request by Apfel**

### CODE REVIEW

File: src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java [hunk 1]
```diff
@@ -103,13 +103,19 @@ else if(tList.size()>1)
         }
 
         tc.getMessageById(id).queue(msg -> {
+            if(msg.getContentRaw().isEmpty())
+            {
+                event.replyWarning("The message i got is empty.");
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java]: I dont really like the message.

File: src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java [hunk 2]
```diff
@@ -128,7 +134,7 @@ else if(!(image==null))
             builder.setAuthor(author.getName()+"#"+author.getDiscriminator(), null, author.getEffectiveAvatarUrl());
             builder.setColor(tc.getGuild().getMember(author)==null?null:tc.getGuild().getMember(author).getColor());
             builder.setDescription(sb.toString());
-            builder.addField("Link", msg.getJumpUrl(), false);
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java]: This is broken on PC.

Comment (Apfel) [reply src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java]: i doubt it

Comment (Artuto) [reply src/main/java/me/artuto/endless/commands/tools/QuoteCmd.java]: Well, it is

**Comment by Apfel** [comment]
Fixed

**Pull request merged**